lib/msal-common/src/client/SPAClient.ts

```diff
@@ -69,7 +70,7 @@ export class SPAClient extends BaseClient {
      */
     private async createUrl(request: AuthenticationParameters, isLoginCall: boolean): Promise<string> {
         // Initialize authority or use default, and perform discovery endpoint check.
-        const acquireTokenAuthority = (request && request.authority) ? AuthorityFactory.createInstance(request.authority, this.networkClient) : this.defaultAuthorityInstance;
+        const acquireTokenAuthority = (request && request.authority) ? AuthorityFactory.createInstance(request.authority, this.networkClient) : this.defaultAuthority;
```

Comment:
What is the user passes the same `authority` in `Configuration` and `Request`? We are still ending up creating an instance when not needed. Should we do the aliasing check and get away with that?

Reply:
This is a good point. I have made a note of it in the comments. We should recommend people only pass in authority to client application, and only override in request if it's different. I will leave this as it is, but if we want to further optimize we can add this later on.